code/espurna/config/general.h

```diff
@@ -1079,6 +1079,10 @@
 #define MQTT_TOPIC_OTA              "ota"
 #define MQTT_TOPIC_TELNET_REVERSE   "telnet_reverse"
 
+// KingArt WiFi Curtain Switch
+#define MQTT_TOPIC_CURTAININ        "curtain"
+#define MQTT_TOPIC_CURTAINOUT       "curtain"
```

Comment:
What I also meant was to use just one `CURTAIN` topic define here.

Reply:
Changed